common/remote_object.go

```diff
@@ -189,25 +193,71 @@ func valueFromRemoteObject(ctx context.Context, robj *cdpruntime.RemoteObject) (
 	return k6ext.Runtime(ctx).ToValue(val), err
 }
 
-func handleParseRemoteObjectErr(ctx context.Context, err error, logger *logrus.Entry) {
-	var (
-		ooe *objectOverflowError
-		ope *objectPropertyParseError
-	)
-	var merr *multiError
-	if !errors.As(err, &merr) {
-		// If this panics it's a bug :)
-		k6ext.Panic(ctx, "parsing remote object value: %w", err)
-	}
-	for _, e := range merr.Errors {
-		switch {
-		case errors.As(e, &ooe):
-			logger.Warn(ooe)
-		case errors.As(e, &ope):
-			logger.WithError(ope).Error()
-		default:
-			// If this panics it's a bug :)
-			k6ext.Panic(ctx, "parsing remote object value: %w", e)
+func parseConsoleRemoteObjectPreview(logger *log.Logger, op *cdpruntime.ObjectPreview) string {
+	obj := make(map[string]string)
+	if op.Overflow {
+		logger.Warnf("parseConsoleRemoteObjectPreview", "object is too large and will be parsed partially")
```

Comment:
Shouldn't this return an error and let the callers decide whether to log or what to do with that error?

Reply:
This is a good candidate for a warning over an error that is returned back to the caller. The warning notifies the user that the module was unable to print the whole output from `console.log`, but we don't want to end the process here since the module will still parse and print some of the object in the lines below.

Reply:
Sorry for my confusing suggestion. I wasn't suggesting to return an error to the user. I'm with you there 👍 I mean to return an error from this function to the caller function chains, and then they can decide whether to log it as a warning. This can make these methods more testable.

Reply:
If we returned an error at this point, then we wouldn't complete the parsing of the object. Thinking about it a bit more now, I think a warn is probably not quite right here, it should be an info. It's telling the user that the module didn't receive the whole object, just part of it, and so only the part that it received will be displayed.

Reply:
Hmm, I see your point. That was why we had included multi-errors here. Yes, it could be info, too.

Reply:
Update log from warn to info in a13e10dbbc9da518059c3d3f6c150ec8b1d0a85e